docs/storage/petalibrary/onedrive.md

```diff
@@ -0,0 +1,88 @@
+
+#### Step 3: Open CURC (left pane)
+In the Collection search (left pane) enter: CU Boulder Research Computing
+
+* You'll be asked to log in using your CURC Credentials - after clicking Authenticate you should receive a Duo push on your Duo device which you will need to confirm
+
+![Globus CURC Authentication](CURCcollection.png)
+
+* Once Authenticated you should see your CURC home directory - to access Petalibrary click "Up One Directory" and then select /pl
+
+![Globus CURC Collection](Globus_CURC_Collection.png)
+
+#### Step 4: Select files/folders from OneDrive to be transferred and initiate the transfer
+* In the left pane, select the file or folder you'd like to transfer and click Start - this will initiate a Globus Transfer job
+
+![Globus Initiate Transfer](Globus_Intitiate_Transfer.png)
+
+* View progress of your transfer under the Activity tab in the Globus App
+
+![Globus Activity](Globus_Activity_GDrive.png)
+
+### Using RClone
+
+#### Step 1: download/install rclone and make sure you can run it
+
+* Download the latest version from [https://rclone.org/downloads/](https://rclone.org/downloads/) and install per the developer's documentation.
+* To test, run `rclone --version` at your prompt; if the software is installed a version number will be reported back to you, similar to:
+
+```
+$ rclone --version
+rclone v1.54.1-DEV
+- os/arch: freebsd/amd64
+- go version: go1.16.2
+```
+
+#### Step 2: configure OneDrive remote configuration
+
+_Note: the steps below are also outlined in the [Rclone Documentation for OneDrive](https://rclone.org/onedrive/). We only list the critical steps. If a step is left out, assume you can select the default setting._
+
+* Type `rclone config` to create a new profile for transferring files between OneDrive and PetaLibrary
+* When prompted for whether to configure a “new” or “existing” profile type `n` for "new remote"
+* When prompted to name the new profile provide any descriptive name you like (e.g., `onedrive_johndoe_cu`)
+* When prompted for the type of storage to configure, select the number of the option for "Microsoft OneDrive" (e.g., the number is "28" for _rclone_ v1.54.1)
+* When prompted for Client ID leave the field blank
+* When prompted for Client Secret leave the field blank
+* When prompted choose national cloud region for OneDrive, select the number of the option for "Microsoft Cloud Global (global)" (e.g., the number is "1" for _rclone_ v1.54.1)
+* When prompted for "Edit Advanced config?” choose "n" for no (default). 
+* When prompted for "Use Auto config?” choose "n" for no (default). 
+* You will then need to use your local machine terminal to run $rclone authorize "onedrive". This will open a window in your your browser, you may be asked to authenticate to your Microsoft account, and then you will be asked to allow Rclone to access the files in your `onedrive`. Complete this step to grant access.  If successful you'll receive a "success" message. 
```

Comment:
Separate sentences -->This will open a window in your your browser (you may be asked to authenticate to your Microsoft account). You will then be asked to allow Rclone to access the files in your `onedrive`.